To: vim_dev@googlegroups.com
Subject: Patch 7.3.314
Fcc: outbox
From: Bram Moolenaar <Bram@moolenaar.net>
Mime-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit
------------

Patch 7.3.314 (after 7.3.304)
Problem:    Missing parenthesis.
Solution:   Add it. (Benjamin R. Haskell)
Files:	    src/Make_mvc.mak


*** ../vim-7.3.313/src/Make_mvc.mak	2011-09-14 10:49:44.000000000 +0200
--- src/Make_mvc.mak	2011-09-14 18:56:24.000000000 +0200
***************
*** 785,791 ****
  PERL_INC = /I $(PERL_INCDIR)
  PERL_OBJ = $(OUTDIR)\if_perl.obj $(OUTDIR)\if_perlsfio.obj
  XSUBPP = $(PERL)\lib\ExtUtils\xsubpp
! !if exist($(XSUBPP)
  XSUBPP = $(PERL_EXE) $(XSUBPP)
  !else
  XSUBPP = xsubpp
--- 785,791 ----
  PERL_INC = /I $(PERL_INCDIR)
  PERL_OBJ = $(OUTDIR)\if_perl.obj $(OUTDIR)\if_perlsfio.obj
  XSUBPP = $(PERL)\lib\ExtUtils\xsubpp
! !if exist($(XSUBPP))
  XSUBPP = $(PERL_EXE) $(XSUBPP)
  !else
  XSUBPP = xsubpp
*** ../vim-7.3.313/src/version.c	2011-09-14 18:59:35.000000000 +0200
--- src/version.c	2011-09-14 19:01:12.000000000 +0200
***************
*** 711,712 ****
--- 711,714 ----
  {   /* Add new patch number below this line */
+ /**/
+     314,
  /**/

-- 
Often you're less important than your furniture.  If you think about it, you
can get fired but your furniture stays behind, gainfully employed at the
company that didn't need _you_ anymore.
				(Scott Adams - The Dilbert principle)

 /// Bram Moolenaar -- Bram@Moolenaar.net -- http://www.Moolenaar.net   \\\
///        sponsor Vim, vote for features -- http://www.Vim.org/sponsor/ \\\
\\\  an exciting new programming language -- http://www.Zimbu.org        ///
 \\\            help me help AIDS victims -- http://ICCF-Holland.org    ///
